src/app/course/course-homepage/course-homepage.component.html

```diff
@@ -24,6 +24,14 @@ <h3 class="tui-island__title">Question Bank</h3>
                     >
                         View
                     </a>
+                    <tui-marker-icon *ngIf="user.is_teacher"
+                                     style="float: right"
```

Comment:
Our convention is to not have inline styles. Preferably we use tailwind CSS classes and only if needed we write custom CSS.
Another point to mention is that we prefer to use flex-box over float. You can wrap the view button and this marker in a container and use these classes (`flex`, `justify-between`) us achieve the same result.